docs/.style/homepage.sass

```diff
@@ -85,7 +85,7 @@ header
     transform: translate(-50%, -50%)
 
   #cta-buttons
-    z-index:1
+    z-index: 1
```

Comment:
This again doesn't belong to this PR. Don't touch files that are unrelated to the current goal.